### PR TITLE
DOCSP-42626 mongosh 2.3.0 release notes

### PR DESCRIPTION
**Pull request by jmd-mongo**
## DESCRIPTION

DOCSP-42626 mongosh 2.3.0 release notes

## STAGING

https://preview-mongodbjmdmongo.gatsbyjs.io/mongodb-shell/DOCSP-42626/changelog/#v2.3.0

## JIRA

https://jira.mongodb.org/browse/DOCSP-42626

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66bf97748e811ae6727e27ff

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/changelog.txt [hunk 1]
```diff
@@ -12,6 +12,25 @@ Release Notes
...
+
+New features released in this version:
+
+- :issue:`MONGOSH-1550` - Support for Range GA
+
+Bug fixes in this release:
+
+- :issue:`MONGOSH-1136` - mongosh should use stderr for password prompt
+- :issue:`MONGOSH-1425` - Expand fallback condition for $collStats to command form to account for ADF
+- :issue:`MONGOSH-1820` - 'fields' option is not working as expected in findAndModify
```
Comment (ianf-mongodb) [inline source/changelog.txt:28]:
```suggestion
- :issue:`MONGOSH-1820` - ``fields`` option is not working as expected in findAndModify
```

File: source/changelog.txt [hunk 2]
```diff
@@ -12,6 +12,25 @@ Release Notes
...
    :class: singlecol
 
+v2.3.0
+------
+
+*Released August 16, 2024*
+
+New features released in this version:
+
+- :issue:`MONGOSH-1550` - Support for Range GA
```
Comment (ianf-mongodb) [inline source/changelog.txt:22]: **Question**

Can you add a bit more detail here? I am afraid people will not know what Range GA is reading this off, without a link or additional content. 

```suggestion
- :issue:`MONGOSH-1550` - Support for Range GA
```

File: source/changelog.txt [hunk 3]
```diff
@@ -12,6 +12,25 @@ Release Notes
...
+
+Bug fixes in this release:
+
+- :issue:`MONGOSH-1136` - mongosh should use stderr for password prompt
+- :issue:`MONGOSH-1425` - Expand fallback condition for $collStats to command form to account for ADF
```
Comment (ianf-mongodb) [inline source/changelog.txt:27]:
```suggestion
- :issue:`MONGOSH-1425` - Expand fallback condition for ``$collStats`` to command form to account for ADF
```

File: source/changelog.txt [hunk 4]
```diff
@@ -12,6 +12,25 @@ Release Notes
...
+- :issue:`MONGOSH-1820` - 'fields' option is not working as expected in findAndModify
+- :issue:`MONGOSH-1858` - Investigate proxy e2e test failure on s390x
```
Comment (ianf-mongodb) [inline source/changelog.txt:29]: The engineering team / @addaleax might be able to answer this, but this seems like an internal ticket and we can likely remove from public RN's.

```suggestion

```

**Review by ianf-mongodb — COMMENTED** [review]
Thanks @jmd-mongo I had a few suggestions for these release notes.

File: source/changelog.txt (hunk 1 above)
Comment (jmd-mongo) [reply source/changelog.txt]: Updated.

File: source/changelog.txt (hunk 2 above)
Comment (jmd-mongo) [reply source/changelog.txt]: Updated.

File: source/changelog.txt (hunk 3 above)
Comment (jmd-mongo) [reply source/changelog.txt]: Updated.

File: source/changelog.txt (hunk 4 above)
Comment (jmd-mongo) [reply source/changelog.txt]: Removed for now. Thanks!

File: source/changelog.txt [hunk 5]
```diff
@@ -12,6 +12,24 @@ Release Notes
...
    :class: singlecol
 
+v2.3.0
+------
+
+*Released August 16, 2024*
+
+New features released in this version:
+
+- :issue:`MONGOSH-1550` - Adds support for Queryable Encryption Range queries
```
Comment (addaleax) [inline source/changelog.txt:22]: We probably at least also want to mention:
- Support for proxies in environment variables
- The removal of --tlsUseSystemCA
- That this removes support for the Range Preview version
- That this removes support for out-of-the-box support for automatic encryption on deprecated operating systems
- The new command line flags added for OIDC support

**Review by ianf-mongodb — APPROVED** [review]
Thanks @jmd-mongo LGTM at this point.

File: source/changelog.txt (hunk 5 above)
Comment (jmd-mongo) [reply source/changelog.txt]: I've updated the features list -- thanks! What issue does "removes support for out-of-the-box support for automatic encryption on deprecated Linux operating systems" correspond to?

Thank you!

**Review by jmd-mongo — COMMENTED** [review]
Thanks for your review, @addaleax! Could you give this one another look when you have a moment?

Thanks,
Joe

**Review by lerouxb — APPROVED** [review]

File: source/changelog.txt (hunk 5 above)
Comment (lerouxb) [reply source/changelog.txt]: > What issue does "removes support for out-of-the-box support for automatic encryption on deprecated Linux operating systems" correspond to?

I'm no expert but if I'd have to guess it is probably a side-effect of bumping the node-driver or something like libmongocrypt.

**Review by jmd-mongo — COMMENTED** [review]
Thanks for noting this, @jeff-allen-mongo! I've updated the version in snooty.toml.

Thanks!
Joe

**Pull request merged**